core/webengine.py

```diff
@@ -249,6 +253,12 @@ def eventFilter(self, obj, event):
                 event.accept()
                 return True
 
+            elif event.button() == MOUSE_WHEEL_BUTTON:
+                modifiers = QApplication.keyboardModifiers()
+                if modifiers == Qt.ControlModifier:
+                    self.open_url_new_buffer_other_window(self.url_hovered)
+                    return True
+
         if event.type() == QEvent.Wheel:
```

Comment:
Why not use Ctrl + Button?
Ctrl + button is more general for user.

Reply:
Currently, Ctrl+Button and MouseWheel will open links in new buffers but in the same window as the current page.
Do you think it's ok to override this feature?

Reply:
On second thought, I agree with you and change the code to make both Ctrl+LeftButton and WheelButton to open links in new window.